src/core/searcher.rs

```diff
@@ -49,6 +103,11 @@ impl Searcher {
         &self.index
     }
 
+    /// [SearcherIndexGeneration] which identifies the index data accessed by this generation of `Searcher`.
+    pub fn index_generation(&self) -> &SearcherGeneration {
```

Comment:
I think now you can call this fn just `generation()` + the doc reference needs a fix